presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PackageApiUtils.java

```diff
@@ -69,4 +72,35 @@ public static boolean isRunning()
     {
         return isInstalled() && executeCommand("service", "presto", "status") == 0;
     }
+
+    public static void untarFile(File configTar, Path destinationDir)
+            throws PrestoManagerException
+    {
+        int untarResult = executeCommand("tar", "-xzvf", configTar.getAbsolutePath(), "-C", destinationDir.toAbsolutePath().toString());
+        if (untarResult != 0) {
+            throw new PrestoManagerException(format("Failed to untar the file: %s", configTar), untarResult);
+        }
+    }
+
+    public static void deleteDirContents(Path directory)
+            throws PrestoManagerException
+    {
+        try {
+            deleteDirectoryContents(directory);
+        }
+        catch (IOException e) {
+            throw new PrestoManagerException(format("Failed to delete contents of the directory: %s", directory.toString()), e);
+        }
+    }
+
+    public static void createDir(Path directory)
+            throws PrestoManagerException
+    {
+        try {
+            createDirectories(directory);
+        }
+        catch (IOException e) {
```

Comment:
Ditto. explicitly catch `FileAlreadyExist` & `IO` to provide more useful information.